network_ips.go

```diff
@@ -16,6 +17,14 @@ type LinodeIPAssignment struct {
 	LinodeID int    `json:"linode_id"`
 }
 
+type LinodeReserveIPOptions struct {
```

Comment:
nitpick: Could this be renamed to `AllocateReserveIPOptions`?

Reply:
Yes! I will make the change now